dev/com.ibm.ws.jakartaee.mail.2.0/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
new projects should be named `io.openliberty...` rather than `com.ibm.ws...`.
I expect this one should be `io.openliberty.jakarta.mail.2.0`

Reply:
It would appear that you already have a project with that name ... but I think the one you already have should be `io.openliberty.mail.2.0` be cause it is the Open Liberty mail API, not the Jakarta API.